aten/src/ATen/native/mkldnn/Normalization.cpp

```diff
@@ -39,38 +53,87 @@ std::tuple<Tensor, Tensor, Tensor> mkldnn_batch_norm(
     bool train,
     double momentum,
     double eps) {
+  TORCH_CHECK(input.dim() == 4 || input.dim() == 5,
```

Comment:
No test case for 3d

Reply:
3d operators will be enable at next step, including conv3d, pool3d, and other mkldnn operators.